README.md

```diff
@@ -97,6 +97,24 @@ To use a model to classify a given bug, you can run `python -m scripts.bug_class
 
 **testing** To use the model to classify a given bug, you can run `python -m scripts.bug_classifier defect --bug-id ID_OF_A_BUG_FROM_BUGZILLA`.
 
+### Training on Taskcluster (Mozilla's CI platform)
+
+You could run the model training task on the CI. To do this, simply include `Train on Taskcluster: <model name>` in the Pull request's description.
```

Comment:
I think all instances should be either "Pull Request" or "pull request". I prefer the second.

Reply:
I think we can go with pull request. Looks neater.